server/server.go

```diff
@@ -111,6 +111,16 @@ type Server struct {
 	mapMu               sync.RWMutex
 	creatingCollections map[string]bool
 	iRODSConnections    map[string]int
+	iRodsTracker        SlackConfig
+}
+
+// SlackConfig is used for slack messages that debounce, such as clients
+// uploading and iRODS connections.
+type SlackConfig struct {
+	slacker  set.Slacker
+	debounce time.Duration
```

Comment:
Probably better to rename this something like `debounceTimeout`, or similar, to indicate it's usage elsewhere.